modules/claims_api/app/controllers/claims_api/v2/veterans/disability_compensation_controller.rb

```diff
@@ -20,14 +20,13 @@ class DisabilityCompensationController < ClaimsApi::V2::Veterans::Base
         before_action :shared_validation, :file_number_check, only: %i[submit validate]
 
         def submit # rubocop:disable Metrics/MethodLength
-          auto_claim = ClaimsApi::AutoEstablishedClaim.create(
+          auto_claim = ClaimsApi::AutoEstablishedClaim.first_or_create(
             status: ClaimsApi::AutoEstablishedClaim::PENDING,
             auth_headers:,
             form_data: form_attributes,
             cid: token.payload['cid'],
             veteran_icn: target_veteran.mpi.icn
           )
```

Comment:
Neither the create method or [this search for an existing claim work working on several of my test cases](https://github.com/department-of-veterans-affairs/vets-api/blob/API-29607-handle-null-values/modules/claims_api/app/controllers/claims_api/v2/veterans/disability_compensation_controller.rb#L33), so I wanted to discuss the idea or find or create. @FonzMP

Reply:
can you elaborate a bit? I was receiving the expected behavior from this section. Curious as to what you were experiencing. what were you getting from the create that made you want to switch to first_or_create? and above you say find or create but here you have first or create?